_install-and-configure/upgrade-opensearch/rolling-upgrade.md

```diff
@@ -0,0 +1,196 @@
+{:.note}
+
+## Preparing to upgrade
+
+Review [Upgrading OpenSearch]({{site.url}}{{site.baseurl}}/upgrade-opensearch/index/) for recommendations about backing up your configuration files and creating a snapshot of the cluster state and indexes before you make any changes to your OpenSearch cluster.
+
+**Important:** OpenSearch nodes cannot be downgraded. If you need to revert the upgrade, then you will need to perform a fresh installation of OpenSearch and restore the cluster from a snapshot. Take a snapshot and store it in a remote repository before beginning the upgrade procedure.
+{: .important}
+
+## Upgrade steps
+
+1. Verify the health of your OpenSearch cluster before you begin. You should resolve any index or shard allocation issues prior to upgrading to ensure that your data is preserved. A status of **green** indicates that all primary and replica shards are allocated. See [Cluster health]({{site.url}}{{site.baseurl}}/api-reference/cluster-api/cluster-health/) for more information.
+   ```bash
+   curl "http://localhost:9201/_cluster/health?pretty"
+   ```
+   Sample output:
+   ```json
+   {
+       "cluster_name":"opensearch-dev-cluster",
+       "status":"green",
+       "timed_out":false,
+       "number_of_nodes":4,
+       "number_of_data_nodes":4,
+       "active_primary_shards":1,
+       "active_shards":4,
+       "relocating_shards":0,
+       "initializing_shards":0,
+       "unassigned_shards":0,
+       "delayed_unassigned_shards":0,
+       "number_of_pending_tasks":0,
+       "number_of_in_flight_fetch":0,
+       "task_max_waiting_in_queue_millis":0,
+       "active_shards_percent_as_number":100.0
+   }
+   ```
+1. Disable shard replication to prevent shard replicas from being created while nodes are being taken offline. This stops the movement of Lucene index segments on nodes in your cluster.
+   ```bash
+   curl -X PUT "http://localhost:9201/_cluster/settings?pretty" -H 'Content-type: application/json' -d'{"persistent":{"cluster.routing.allocation.enable":"primaries"}}'
+   ```
+   Sample output:
+   ```json
+   {
+     "acknowledged" : true,
+     "persistent" : {
+       "cluster" : {
+         "routing" : {
+           "allocation" : {
+             "enable" : "primaries"
+           }
+         }
+       }
+     },
+     "transient" : { }
+   }
+   ```
+1. Perform a flush operation on the cluster to commit transaction log entries to the Lucene index.
+   ```bash
+   curl -X POST "http://localhost:9201/_flush?pretty"
+   ```
+   Sample output:
+   ```json
+   {
+     "_shards" : {
+       "total" : 4,
+       "successful" : 4,
+       "failed" : 0
+     }
+   }
+   ```
+1. Review your cluster and identify the first node to upgrade. Eligible cluster manager nodes should be upgraded last because OpenSearch nodes can join a cluster with manager nodes running an older version, but they cannot join a cluster with all manager nodes running a newer version.
```

Comment:
```suggestion
1. Review your cluster and identify the first node to be upgraded. Eligible cluster manager nodes should be upgraded last because OpenSearch nodes can join a cluster with manager nodes running an older version, but they cannot join a cluster with all manager nodes running a newer version.
```